sphinx/ext/autosummary/generate.py

```diff
@@ -151,6 +153,16 @@ def get_members(obj: Any, types: Set[str], include_public: List[str] = [],
                   if x in include_public or not x.startswith('_')]
         return public, items
 
+    def get_modules(obj: Any, include_public: List[str] = []) -> Tuple[List[str], List[str]]:
```

Comment:
`include_public` is not used. It should be removed. Same option in `get_members()` is only used for `__init__()` method. No reason to add such option to here.